src/tallies/tally_header.F90

```diff
@@ -388,6 +388,7 @@ subroutine configure_tallies()
 
     do i = 1, n_tallies
       call tallies(i) % obj % allocate_results()
+      tallies(i) % obj % active = .false.
```

Comment:
If instead of setting all tallies to be inactive during `simulation_init`, we set them to inactive during `simulation_finalize`, this would allow us to retain the ability to change the active flag before `simulation_init`. Looking at my code for equilibrium Xenon calculations, that's exactly what I was doing. Although I could rearrange to call `simulation_init` first, I think it's nice to not have to rely on that. What do you think?

Reply:
Good idea!  I've moved it to `simulation_finalize` now.